lib/parser.js

```diff
@@ -68,6 +68,9 @@ var selectorTokens = [
 var pseudoSelectors = [
   // Logical Combinations
     'matches'
+  , 'is'
```

Comment:
I think we can update all valid Selectors Level 4 pseudo-classes ? 
https://www.w3.org/TR/selectors-4

Reply:
Probably will do that in a later PR, as this will require careful reading through the spec which I don't have time for right now.